Sources/Task.swift

```diff
@@ -47,12 +47,14 @@ public struct Task {
 }
 
 private extension String {
+	static let whitespaceRegularExpression = try! NSRegularExpression(pattern: "\\s")
```

Comment:
I'd make this explicitly `private`. If the visibility of the extension is changed, we still want this to remain an implementation detail.